faasmcli/faasmcli/util/policy.py

```diff
@@ -0,0 +1,218 @@
+def parse_verify(verify):
```

Comment:
The level of commenting in this PR is quite inconsistent. In the relatively simple `get_file_hash_digest` function there's a comment on every single line, whereas this whole file containing the complicated policy logic has no comments at all. Can you add some?